src/orchestrator/orchestrator.rs

```diff
@@ -265,12 +267,26 @@ impl Borrow<ArtifactPath> for ProducedArtifact {
 
 impl<'a> Orchestrator<'a> {
     pub async fn run(self, output: &mut Vec<ArtifactPath>) -> Result<HashMap<Uuid, Error>> {
-        let (results, errors) = self.run_tree().await?;
+        let token = CancellationToken::new();
```

Comment:
Nit: This is a very generic variable name...

Reply:
I was following the examples on the tokio.rs site (https://tokio.rs/tokio/topics/shutdown)
_They also just called it `token` 😨_ 
But you're right. How about `cancellation_token`?

Reply:
Well, duh, for such short examples the variable names don't matter much and the examples are there specifically to demonstrate the concept :D
That doesn't mean you should reuse those names for code in real projects (where `token` could be any kind of token) ;)

`cancellation_token` should be fine :)